portal-ui/src/screens/Console/Users/DeleteUser.tsx

```diff
@@ -78,14 +128,42 @@ const DeleteUser = ({
       onConfirm={onConfirmDelete}
       onClose={onClose}
       confirmationContent={
-        <DialogContentText>
-          Are you sure you want to delete the following {selectedUsers.length}{" "}
-          user{selectedUsers.length > 1 ? "s?" : "?"}
-          <b>{renderUsers}</b>
+        <DialogContentText>  
+          
+        {hasSA ? 
+        <>
```

Comment:
Please use explicit Fragment
```suggestion
        <Fragment>
```